apps/modernization-ui/src/validation/entry/validPageNameRule.ts

```diff
@@ -0,0 +1,14 @@
+import { maxLengthRule } from './maxLengthRule';
+
+/**
+ * Creates a standardized react-hook-form rule for the entry of name values.
+ */
+const validPageNameRule = {
+    ...maxLengthRule(),
+    pattern: {
+        value: /^[\w*()_+\-=;:/.,\d\s]+$/gm,
+        message: 'Only alphabets, digits, and some special character allowed such as ( ) _ + - = ; : / . ,'
```

Comment:
Nit/question (non-blocking): what do you think about: `Valid characters are A-Z, a-z, 0-9, or ()_+-=;:/.,`